docs/src/content/docs/fr/components/asides.mdx

```diff
@@ -0,0 +1,158 @@
+---
+
+import { Aside } from '@astrojs/starlight/components';
+
+Pour afficher des informations secondaires à côté du contenu principal d'une page, utilisez le composant `<Aside>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Aside slot="aperçu">
+Incluez des informations supplémentaires non essentielles dans un `<Aside>`.
+</Aside>
+
+</Preview>
+
+## Import
+
+```tsx
+import { Aside } from '@astrojs/starlight/components';
+```
+
+## Utilisation
+
+Affichez un encart à l'aide du composant `<Aside>`.
+
+Un `<Aside>` peut avoir un attribut facultatif [`type`](#type), qui contrôle la couleur, l'icône et le titre par défaut de l'encart.
+
+<Preview>
+
+````mdx
+import { Aside } from '@astrojs/starlight/components';
+
+<Aside>Un peu de contenu en encart.</Aside>
+
+<Aside type="caution">Quelques mises en garde.</Aside>
+
+<Aside type="tip">
+
+	D'autres contenus sont également pris en charge dans les encarts.
+
+```js
+// Un extrait de code, par exemple.
+```
+
+</Aside>
+
+<Aside type="danger">Ne communiquez votre mot de passe à personne.</Aside>
+````
+
+<Fragment slot="markdoc">
+
+````markdoc
+{% aside %}
+Un peu de contenu en encart.
+{% /aside %}
+
+{% aside type="caution" %}
+Quelques mises en garde.
+{% /aside %}
+
+{% aside type="tip" %}
+D'autres contenus sont également pris en charge dans les encarts.
+
+```js
+// Un extrait de code, par exemple.
+```
+{% /aside %}
+
+{% aside type="danger" %}
+Ne communiquez votre mot de passe à personne.
+{% /aside %}
+````
+
+</Fragment>
+
+<Fragment slot="preview">
+
+    <Aside>Un peu de contenu en encart.</Aside>
+
+    <Aside type="caution">Quelques mises en garde.</Aside>
+
+    <Aside type="tip">
+
+	D'autres contenus sont également pris en charge dans les encarts.
+
+    ```js
+    // Un extrait de code, par exemple.
+    ```
+
+    </Aside>
+
+    <Aside type="danger">Ne communiquez votre mot de passe à personne.</Aside>
+
+</Fragment>
+
+</Preview>
+
+Starlight fournit également une syntaxe personnalisée pour afficher les encarts dans Markdown et MDX comme alternative au composant `<Aside>`.
+Voir le guide [« Authoring Content in Markdown »](/fr/guides/authoring-content/#encarts) pour plus de détails sur la syntaxe personnalisée.
+
+### Utilisation de titres personnalisés
+
+Remplacez les titres par défaut des encarts en utilisant l'attribut [`title`](#title).
+
+<Preview>
+
+```mdx 'title="Watch out!"'
+import { Aside } from '@astrojs/starlight/components';
+
+<Aside type="caution" title="Attention !">
+	Un avertissement est mis de côté *avec* un titre personnalisé.
+</Aside>
+```
+
+<Fragment slot="markdoc">
+
+```markdoc 'title="Attention !"'
+{% aside type="caution" title="Attention !" %}
+Un avertissement est mis de côté *avec* un titre personnalisé.
+{% /aside %}
+```
+
+</Fragment>
+
+<Aside slot="preview" type="caution" title="Attention !">
+	Un avertissement est mis de côté *avec* un titre personnalisé.
+</Aside>
+
+</Preview>
+
+## `<Aside>` Props
```

Comment:
```suggestion
## Props de `<Aside>`
```

I've been using this form so far in French translation to avoid keeping the English one which would be incorrect in French.